### PR TITLE
Update Italian translation

### PR DESCRIPTION
**Pull request by atilluF**

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -59,25 +59,25 @@
...
     <string name="edit">Modifica</string>
-    <string name="start_radio">Apri la radio</string>
+    <string name="start_radio">Apri radio</string>
     <string name="play">Riproduci</string>
     <string name="play_next">Riproduci come successiva</string>
-    <string name="add_to_queue">Aggiungi alla coda</string>
-    <string name="add_to_library">Aggiungi alla libreria</string>
-    <string name="remove_from_library">Rimuovi dalla libreria</string>
+    <string name="add_to_queue">Metti in coda</string>
+    <string name="add_to_library">Aggiungi a libreria</string>
```
Comment (airon90) [inline app/src/main/res/values-it/strings.xml:66]: Why did you remove the article?

Comment (airon90) [reply app/src/main/res/values-it/strings.xml:66]: Same on string 67

**Comment by z-huang** [comment]
Thanks!
